tests/emodulus_calibration/test_emodulus_calibration.py

```diff
@@ -0,0 +1,41 @@
+from lebedigital.calibration.calibrationWorkflow import esimate_Youngs_modulus
+from lebedigital.calibration.utils import query_KG
+from probeye.ontology.knowledge_graph_import import import_parameter_samples
+
+import pytest
+import numpy as np
+import os
+from pathlib import Path
+
+def test_emodulus_calibration():
+
+    # query knowledge graph. Note KG team needs to provide this.
+    print("Calibration test started")
+    output = query_KG(
+        path=os.path.dirname(Path(__file__)), exp_name="Wolf 8.2 Probe 1.csv"
+    )
+    E_samples, post_pred = esimate_Youngs_modulus(
+        experimental_data=output, calibration_metadata={"E_loc": 100, "E_scale": 100},
+        calibrated_data_path=os.path.dirname(Path(__file__)), mode='cheap'
+    )
+    # checking if the KG file for the calibration is created
+    assert os.path.exists(os.path.join(os.path.dirname(Path(__file__), 'calibrationWorkflowWolf 8.2 Probe 1.csv.owl'))) == True
+
+    # checking for the calibrated E modulus and the posterior predictive
+    assert np.mean(post_pred) == pytest.approx(95, rel=50)
+
+    assert np.mean(E_samples) == pytest.approx(95E03, rel=50e03)
```

Comment:
There is something wrong here, the E-modulus of these tests should not be 95e3Mpa. That is too large.

Reply:
I think this might be the result of using the specimen length (around 300mm) instead of the gauge length (100mm) in some places.

Reply:
@joergfunger alright, then in what range it should be? I remember asking this multiple time that what is the expected range  then and not getting any input from domain experts/ or what is the standard E modulus from ususal methods (maybe standards).

Reply:
Its expected to be around 30000 N/m2 (or Pa). The value is about 3 times too large. Thats, why I think it might have something todo, with applying the wrong length somewhere.